File: tests/python/test_project.py
import unittest

from vio_tools import default_seed, project_name


class ProjectTest(unittest.TestCase):
    def test_project_name(self) -> None:
        self.assertEqual(project_name(), "Vio Autonomous WALL-E")

    def test_default_seed(self) -> None:
        self.assertEqual(default_seed(), 42)


if __name__ == "__main__":
    unittest.main()

File: tests/cpp/test_project.cpp
#include "vio/project.hpp"
#include <iostream>

int main() {
    if (vio::project_name() != "VIO Autonomous WALL-E") {
        std::cerr << "Unexpected project name\n";
        return 1;
    }

    if (vio::default_seed() != 42) {
        std::cerr << "unexpected default seed\n";
        return 1;
    }

    std::cout << "test passed";
    return 0;
}
